### ISSUE TITLE
ERROR: Error executing transaction, executing rollback. Inner stack: Error: Note title must not be empty

### ISSUE THREAD

**Issue by bonedaddy** [zadam/trilium-web-clipper#16]
# Details

Whenever I try saving pages with the clipper on firefox I keep getting this error. I was able to save one page successfully, but now keep getting errors.

* [URL I was able to save](https://developers.redhat.com/blog/2020/03/26/static-analysis-in-gcc-10/)
* [URL I'm not able to save](https://www.cs.swarthmore.edu/~newhall/unixhelp/C_arrays.html)

# Error

```
21:30:43.928 INFO  POST /api/clipper/notes
21:30:43.931 INFO  ERROR: Error executing transaction, executing rollback. Inner stack: Error: Note title must not be empty
    at Object.createNewNote (/home/solidity/Programs/trilium-linux-x64-server/src/services/notes.js:105:15)
    at async createNote (/home/solidity/Programs/trilium-linux-x64-server/src/routes/api/clipper.js:74:20)
    at async /home/solidity/Programs/trilium-linux-x64-server/src/routes/routes.js:92:32
    at async /home/solidity/Programs/trilium-linux-x64-server/src/services/sql.js:220:19
Outside stack: Error
    at Object.transactional (/home/solidity/Programs/trilium-linux-x64-server/src/services/sql.js:211:23)
    at /home/solidity/Programs/trilium-linux-x64-server/src/routes/routes.js:91:38
    at /home/solidity/Programs/trilium-linux-x64-server/node_modules/cls-hooked/context.js:118:13
    at Namespace.run (/home/solidity/Programs/trilium-linux-x64-server/node_modules/cls-hooked/context.js:97:5)
    at Namespace.runAndReturn (/home/solidity/Programs/trilium-linux-x64-server/node_modules/cls-hooked/context.js:117:8)
    at Object.init (/home/solidity/Programs/trilium-linux-x64-server/src/services/cls.js:5:28)
    at /home/solidity/Programs/trilium-linux-x64-server/src/routes/routes.js:85:38
    at Layer.handle [as handle_request] (/home/solidity/Programs/trilium-linux-x64-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/solidity/Programs/trilium-linux-x64-server/node_modules/express/lib/router/route.js:137:13)
    at checkToken (/home/solidity/Programs/trilium-linux-x64-server/src/services/auth.js:67:9)
21:30:43.931 INFO  Creating clipped note from https://www.cs.swarthmore.edu/~newhall/unixhelp/C_arrays.html
21:30:43.931 INFO  ERROR: post /api/clipper/notes threw exception: Error: Note title must not be empty
    at Object.createNewNote (/home/solidity/Programs/trilium-linux-x64-server/src/services/notes.js:105:15)
    at async createNote (/home/solidity/Programs/trilium-linux-x64-server/src/routes/api/clipper.js:74:20)
    at async /home/solidity/Programs/trilium-linux-x64-server/src/routes/routes.js:92:32
    at async /home/solidity/Programs/trilium-linux-x64-server/src/services/sql.js:220:19
21:30:45.970 INFO  POST /api/recent-notes
21:31:27.156 INFO  GET /api/clipper/handshake
21:31:27.159 INFO  GET /api/clipper/handshake rejected with 401 Not authorized
21:31:27.215 INFO  GET /api/clipper/handshake

```

**Comment by zadam** [comment]
Hi, thanks for reporting the error. This was fixed on the server side and will be released in 0.43.2

**Comment by bonedaddy** [comment]
can confirm 0.43.2 fixed the issue
